api/src/pcapi/routes/pro/collective_offers.py

```diff
@@ -206,6 +206,10 @@ def edit_collective_offer(
         offers_api.update_collective_offer(offer_id=dehumanized_id, new_values=body.dict(exclude_unset=True))
     except offers_exceptions.SubcategoryNotEligibleForEducationalOffer:
         raise ApiErrors({"subcategoryId": "this subcategory is not educational"}, 400)
+    except offers_exceptions.OffererOfVenueDontMatchOfferer:
+        raise ApiErrors({"offerer": "new venue need to have same offerer"}, 403)
```

Comment:
```suggestion
        raise ApiErrors({"venueId": "new venue need to have same offerer"}, 403)
```
c'est le champ venueId qui est fautif